services/twitter/twitter.tester.js

```diff
@@ -15,25 +15,13 @@ t.create('Followers')
     message: isMetric,
   })
 
-t.create('Followers - Custom Label')
-  .get('/follow/shields_io.json?label=Follow')
-  .expectBadge({
-    label: 'Follow',
-    message: isMetric,
-  })
-
 t.create('Invalid Username Specified')
   .get('/follow/invalidusernamethatshouldnotexist.json?label=Follow')
   .expectBadge({
     label: 'Follow',
     message: 'invalid user',
   })
 
-t.create('No connection')
-  .get('/follow/shields_io.json?label=Follow')
-  .networkOff()
-  .expectBadge({ label: 'Follow', message: 'inaccessible' })
-
 t.create('URL')
   .get('/url/https/shields.io.json')
   .expectBadge({ label: 'tweet', message: '' })
```

Comment:
This test isn't really testing the most important thing, which is the content of the links array, but this PR isn't making that worse, so I think I'm going to punt the job of improving the `expectBadge()` helper to another issue